### PR TITLE
#216 try to guess zip folder name

### PR DESCRIPTION
**Pull request by douph1**

### CODE REVIEW

File: wix/Build.OpenJDK_generic.cmd [hunk 1]
```diff
@@ -70,6 +70,24 @@ SET PRODUCT_SKU=OpenJDK
 SET PRODUCT_VERSION=%PRODUCT_MAJOR_VERSION%.%PRODUCT_MINOR_VERSION%.%PRODUCT_MAINTENANCE_VERSION%.%PRODUCT_PATCH_VERSION%
 SET ICEDTEAWEB_DIR=.\SourceDir\icedtea-web-image
 
+REM JEP322 alike
+SET JEP322_BASE_NAME="unknown"
+IF %PRODUCT_CATEGORY% EQU jre SET JRE=-jre
+IF %PRODUCT_MAJOR_VERSION% GEQ 10 (
+    IF %PRODUCT_MAJOR_VERSION% EQU 11 SET LTS_VERSION=-LTS
```
Comment (karianna) [inline wix/Build.OpenJDK_generic.cmd:77]: Maybe make 11 in an array and add 17?

File: wix/Build.OpenJDK_generic.cmd [hunk 2]
```diff
@@ -70,6 +70,24 @@ SET PRODUCT_SKU=OpenJDK
...
+    IF DEFINED PRODUCT_PATCH_VERSION (
+    SET PATCH=+%PRODUCT_PATCH_VERSION%
```
Comment (karianna) [inline wix/Build.OpenJDK_generic.cmd:79]: Indent this line?

File: wix/Build.OpenJDK_generic.cmd (hunk 1 above)
Comment (douph1) [reply wix/Build.OpenJDK_generic.cmd:77]: >is this trying to solve the problem of creating installers for the jdk tip version (which is currently 15)?

yes

yes we can add 17 for the LTS flag but I have not seen any artifact follow this JEP322 about LTS flag:

>jdk-11.0.7+10/bin/java -version
openjdk version "11.0.7" 2020-04-14
OpenJDK Runtime Environment AdoptOpenJDK (build 11.0.7+10)
OpenJDK 64-Bit Server VM AdoptOpenJDK (build 11.0.7+10, mixed mode)

LTS flag is assigned to JEP322_NAME var
but as we don't follow JEP322 actually this is just for future need if we want to align to JEP322
This code is useless now 

Did you prefer I remove or comment this code ?

Is there any plan to follow JEP322 ?

And I have not seen any documentation/annonce about 17 will be effectively LTS :) .. roadmap can change again..

File: wix/Build.OpenJDK_generic.cmd (hunk 2 above)
Comment (douph1) [reply wix/Build.OpenJDK_generic.cmd]: done

File: wix/Build.OpenJDK_generic.cmd (hunk 1 above)
Comment (karianna) [reply wix/Build.OpenJDK_generic.cmd:77]: Let's not worry about adding 17 yet, we can do that later

**Comment by M-Davies** [comment]
Pipelines are in progress to test the PR https://github.com/AdoptOpenJDK/openjdk-build/pull/1790#issuecomment-639382545

**Comment by M-Davies** [comment]
@douph1 Looks like it failed with a different error https://ci.adoptopenjdk.net/job/build-scripts/job/release/job/create_installer_windows/4355/console
```
J:\jenkins\workspace\build-scripts\release\create_installer_windows\wix\SourceDir>IF ERRORLEVEL 1 (
ECHO unzip failed  
 EXIT /b 2 
) 

J:\jenkins\workspace\build-scripts\release\create_installer_windows\wix\SourceDir>cd J:\jenkins\workspace\build-scripts\release\create_installer_windows\wix 

J:\jenkins\workspace\build-scripts\release\create_installer_windows\wix>call Build.OpenJDK_generic.cmd 
'ADOPT_NAME' is not recognized as an internal or external command,
operable program or batch file.
Generate OpenJDK setup "hotspot" for "x64" platform "jdk"
****************************************************
First .\SourceDir\OpenJDK15\hotspot\x64\jdk-15.0.0 not exists
Second .\SourceDir\OpenJDK15\hotspot\x64\jdk-15+26 not exists
Third .\SourceDir\OpenJDK15\hotspot\x64\jdk-15.0.0+26 not exists
Source dir used : .\SourceDir\OpenJDK-Latest\hotspot\x64\
PRODUCT_ID: {BE9A971F-8151-434F-BBF8-4B3E1ABAB50A}
SOURCE_TEXT_GUID (without displaying secret UPGRADE_CODE_SEED) : jdk-15-x64-hotspot
Constant PRODUCT_UPGRADE_CODE: {0E729B05-A883-7C61-B47D-FB38DF729B46}
copy "Main.hotspot.wxs"
        1 file(s) copied.
HEAT
heat.exe : error HEAT0319 : The '-out' or '-o' parameter must specify a file path.
Failed to generating Windows Installer XML Source files (.wxs)
```

**Pull request merged**